bundles/net.tourbook/src/net/tourbook/importdata/DialogEasyImportConfig.java

```diff
@@ -144,7 +144,7 @@
    private static final String          CSS_PX                            = "px";                                //$NON-NLS-1$
    private final static String[]        DeviceTypes                       = new String[] {
          Messages.Dialog_ImportConfig_Combo_Device_LocalDevice,
-         Messages.Dialog_ImportConfig_Combo_Device_Dropbox
+         "Dropbox"                                                                                               //$NON-NLS-1$
    };
```

Comment:
The `Dropbox` item is static, when the cloud plugin is not enabled, then Dropbox can currently still be selected. 
-> all items in DeviceType, except the local entry, must be provided from the available plugins, so DeviceType must be filled dynamically

In my first test of this PR I forgot to enable the cloud plugin and I just saw this issue.

Reply:
Fixed in this [PR](https://github.com/wolfgang-ch/mytourbook/pull/191)